package.json

```diff
@@ -8,7 +8,7 @@
     "webpack-defaults": "webpack-defaults",
     "start": "npm run build -- -w",
     "appveyor:test": "npm run test",
-    "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
```

Comment:
Well it's not needed (here) if folks prefer to use a separate `test` folder, but both was are supported and the boilerplate should stay in sync :)